app/models/user.rb

```diff
@@ -61,11 +61,17 @@ def notifications_configured?
     user_notifications.count.positive?
   end
 
+  before_create do
```

Comment:
Is there a better way of doing this? It's just needed for the user controller create action, but a `before_action` wouldn't have worked